doc/orchestrator.rst

```diff
@@ -63,108 +326,207 @@ on that compute host.
 
 |colo-orc|
 
-This deployment is designed for highly performant online inference scenarios where
+Colocated deployment is designed for highly performant online inference scenarios where
 a distributed process (likely MPI processes) are performing inference with
-data local to each process.
+data local to each process. Data produced by these processes and stored in the colocated database
+can be transferred via a SmartRedis client to a standard database to become available for consumption
+by other applications. A tradeoff of colocated deployment is the ability to scale to a large workload.
+Colocated deployment rather benefits small/medium simulations with low latency requirements.
+By hosting the database and simulation on the same compute node, communication time is reduced which
+contributes to quicker processing speeds.
 
 This method is deemed ``locality based inference`` since data is local to each
 process and the ``Orchestrator`` is deployed locally on each compute host where
 the distributed application is running.
 
+-------
+Example
+-------
+In the following example, we provide a demonstration on automating the deployment of
+a colocated Orchestrator using SmartSim from within a Python driver script. Once the colocated database is launched,
+we demonstrate connecting a client to the database from within the application script to transmit and poll data.
 
-To create a colocated model, first, create a ``Model`` instance and then call
-the ``Model.colocate_db_tcp`` or ``Model.colocate_db_uds`` function.
+The example is comprised of two script files:
+
+- The Application Script
+   The example application script is a Python file that contains
```

Comment:
The example application script is a Python file that contains
   instructions to create and connect a SmartRedis
   client to the colocated Orchestrator.

->

   The example application is a Python script that connects a SmartRedis
   client to the colocated Orchestrator.